docs/mixin/dashboards/promscale.json

```diff
@@ -761,9 +761,9 @@
                 "uid": "${DS_PROMETHEUS}"
               },
               "exemplar": true,
-              "expr": "rate(grpc_server_handled_total{grpc_service=~\"opentelemetry.proto.collector.trace.v1.TraceService\",grpc_method=~\"Export\"}[$__rate_interval]) > 0",
+              "expr": "sum(rate(grpc_server_handled_total{grpc_service=~\"opentelemetry.proto.collector.trace.v1.TraceService\",grpc_method=~\"Export\"}[$__rate_interval])) + sum(rate(grpc_server_msg_received_total{grpc_service=\"jaeger.storage.v1.StreamingSpanWriterPlugin\",grpc_method=\"WriteSpanStream\"}[$__rate_interval])) + sum(rate(grpc_server_handled_total{grpc_service=\"jaeger.storage.v1.SpanWriterPlugin\",grpc_method=\"WriteSpan\"}[$__rate_interval]))",
```

Comment:
Can you try below?

```suggestion
              "expr": "rate(grpc_server_msg_received_total{grpc_method=~\"(WriteSpan|WriteSpanStream|Export)\"}[5m])",
```
<img width="1299" alt="Screenshot 2022-10-21 at 11 19 11 AM" src="https://user-images.githubusercontent.com/3874763/197122173-30537b1a-3e7b-4515-b65b-a519d5b395b2.png">